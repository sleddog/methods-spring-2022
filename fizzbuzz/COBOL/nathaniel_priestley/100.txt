Begin shell script execution.
Begin FizzBuzz execution.
 01
 02
Fizz
 04
Buzz
Fizz
Ping
 08
Fizz
Buzz
