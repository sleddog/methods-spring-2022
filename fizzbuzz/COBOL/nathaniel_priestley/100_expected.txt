Begin shell script execution.
Begin FizzBuzz execution.
 01
 02
Fizz
 04
Buzz
Fizz
Ping
 08
Fizz
Buzz
Pong
Fizz
 13
Ping
FizzBuzz
 16
 17
Fizz
 19
Buzz
FizzPing
Pong
 23
Fizz
Buzz
 26
Fizz
Ping
 29
FizzBuzz
 31
 32
FizzPong
 34
BuzzPing
Fizz
 37
 38
Fizz
Buzz
 41
FizzPing
 43
Pong
FizzBuzz
 46
 47
Fizz
Ping
Buzz
Fizz
 52
 53
Fizz
BuzzPong
Ping
Fizz
 58
 59
FizzBuzz
 61
 62
FizzPing
 64
Buzz
FizzPong
 67
 68
Fizz
BuzzPing
 71
Fizz
 73
 74
FizzBuzz
 76
PingPong
Fizz
 79
Buzz
Fizz
 82
 83
FizzPing
Buzz
 86
Fizz
Pong
 89
FizzBuzz
Ping
 92
Fizz
 94
Buzz
Fizz
 97
Ping
FizzPong
Buzz
